src/config/animations.js

```diff
@@ -0,0 +1,2 @@
+export const speed = 100;
+export const friction = 10;
```

Comment:
<p align="center">
<img src = "https://user-images.githubusercontent.com/12476932/28506724-5d716f72-6ffb-11e7-8272-c8750543ecdf.png" width=250>
</p>

I like that `animations` constants are in their own file! But two things I noticed:

1. `duration` isn't in the `animations.js` config file
2. You probably want to export it as an object called `animations` so you can access it with `animations.property` like you did in `mention-area`.

```
export const animations = {
   speed: 100,
   ...
}
```